tests/common/fixtures/ptfhost_utils.py

```diff
@@ -33,7 +33,7 @@
 GARP_SERVICE_PY = 'garp_service.py'
 GARP_SERVICE_CONF_TEMPL = 'garp_service.conf.j2'
 PTF_TEST_PORT_MAP = '/root/ptf_test_port_map.json'
-PROBER_INTERVAL_MS = 1000
+PROBER_INTERVAL_MS = 2000
```

Comment:
Two concerns:
1. this will affect all platforms
2. the toggle disruption threshold should be changed along with this probe interval increase

We might need to figure out why the increasing probe time reduces the testcase failures.